### ISSUE TITLE
Maven central version badge should not be prefixed with 'v'

### ISSUE THREAD

**Issue by onacit** [badges/shields#10574]
### Are you experiencing an issue with...

shields.io

### 🐞 Description

I seems the [Maven Central Version](https://shields.io/badges/maven-central-version) seems prefix `v`. I don't have any clue for any purpose or intention.

e.g.
![Maven Central Version](https://img.shields.io/maven-central/v/com.google.guava/guava)

The `v` doesn't add any value but some confusion.

### 💡 Possible Solution

Remove the `v` prefix from the version value.

**Comment by jNullj** [comment]
Shields.io employs the `renderVersionBadge` function to render versions consistently across the project (horizontal consistency), including maven-central, by always adding the `v` prefix using `addv`. 

We have several options to consider:
1. Change only maven-central to remove the prefix, which would break horizontal consistency and create inconsistency among shields.
2. Remove the prefix in all version renders, improving vertical consistency for this badge.
3. Add an option for users to customize the prefix.

A comprehensive discussion on this topic is already underway in [issue #10437](https://github.com/badges/shields/issues/10437).

IMO option 3 is best while i think the default should stay with the prefix for now until a standard is set.
Let us know what you think and what solutions you had in mind.

**Comment by onacit** [comment]
@jNullj Thank you for the comment. I'm not going to go deep into the issue you mentioned. I agree. Adding an option for users to customize the prefix will help.

**Comment by ryan-williams** [comment]
+1, this isn't specific to Maven Central badges, I'd like to omit the "v" for PyPI and NPM badges as well. It adds no info, and wastes precious horizontal space on mobile.

Obvious solution:
- `messagePrefix` query param, defaults to "v"
- `?messagePrefix=&…` would make it the empty string.

Strange to hard-code the "v" prefix and give no option for overriding.

#10437 seems to be much broader / not obviously related.

**Comment by calebcartwright** [comment]
For historical context I'll also point back to https://github.com/badges/shields/discussions/6135

but regarding 👇 

> https://github.com/badges/shields/issues/10437 seems to be much broader / not obviously related.

#10437 is directly related, the standardized `v` prefix on version badges was discussed there explicitly and at length

e.g. 

https://github.com/badges/shields/issues/10437#issuecomment-2272017327

> Some examples that come to mind of cases where we've opted for horizontal consistency over vertical consistency are things like **where we add prefixes/suffixes (e.g. v on version badges)**, code coverage using different levels of precision (e.g. 92.51% vs 93%), download counts, and pipeline status wording

https://github.com/badges/shields/issues/10437#issuecomment-2272082402

> Yeah all good examples - thanks.
>
> **I feel like always adding v on version badges and decimal precision/rounding are stylistic/presentation concerns**. They don't change the meaning of the value.

etc.

Probably best to read things or at least ask for more direct pointers to related information before pushing back and characterizing something as unrelated

**Comment by robaho** [comment]
> Hi @PyvesB - it says your commit is supposed to address this, but when I add ?prefix=xxxx to the url nothing changes?

actually, it doesn't look like the maven service was changed to use the renderVersionBadge? If I submit a PR will this be accepted, or has this been rejected for some sort of consistency concerns - the issue is unclear regards to this.

**Comment by daniel-rusu** [comment]
In case anyone else struggles with this, I worked around this limitation by storing the latest version in a json file and used the dynamic json shield to fetch the version from the json file and display it correctly.  I also set the color to orange so that it looks like the maven-central shield.  I chose to update the json file manually so I can do some post-release sanity checks before the new version is referenced in the readme, but if you prefer, it's fairly simple to setup a GitHub workflow to automatically update the json file when you create a new GitHub release.

In case it helps, see `latest_version.json` and `README.md` for my solution in https://github.com/daniel-rusu/pods4k
